doc/content/reference/application-packages/lora-cloud-device-and-application-services.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: "LoRa Cloud Device & Application Services"
+title: "LoRa Cloud Modem and Geolocation Services"
```

Comment:
@adriansmares can you please review this section? **Modem & Geolocation Services** shouldn't be the same as **DAS** afaik. I don't know enough about this application package to be sure that this is completely accurate. I see that some things are changed on LoRa Cloud side.. For example, I suppose that a default server URL isn't https://das.loracloud.com anymore?

Reply:
Indeed lots of changes happened on the LoRa Cloud side. They stopped separating what used to be DAS (Device and Application Services) from GLS (Geolocation Services). They are now a unified services called Modem and Geolocation services.